src/bz.js

```diff
@@ -24,34 +25,39 @@ router
         'cf_last_resolved',
         'assigned_to',
         'flags',
+        'whiteboard',
       ].join(','),
     });
     const { bugs } = await fetchJson(`${base}?${query}`, { ttl: 'day' });
+    if (list) {
+      return ctx.body = bugs;
```

Comment:
I'm not 100% familiar with node servers, but is there a reason why we should assign to `ctx.body`? Can we not just return `bugs`?

Reply:
yeah, you have to assign data to ctx.body.